### ISSUE TITLE
Why not a 'Shared' folder for common stuff

### ISSUE THREAD

**Issue by fourctv** [NathanWalker/angular-seed-advanced#58]
Looking at the files inside Client & NativeScript, there are a lot of duplicates.

Why not a Shared, or Library, or ?, as a location for all common shared files between the multiple targets?

Maybe that topic has been discussed already, and I'm coming late. Forgive me if that is the case.

**Comment by NathanWalker** [comment]
Hi @fourctv thanks for the question. The files in `nativescript` folder and `client` folder are different actually. If you are referring to the `node_modules` that get installed in the `nativescript` folder vs. the root of the project; there's definitely duplicate installed modules there. This is for a couple reasons. Right now, the {N} app does not use the seed build and also requires slightly different versions for several libraries so they maintain different sets at times. Eventually this will not be the case and things could be improved there.

Hopefully that answers the question? If not, maybe drop a link to the 2 different areas of duplication you're concerned about. Thanks!

**Comment by fourctv** [comment]
Thanks for the reply Nathan.
Actually my question was based on the contents in NativeScript folder **after** I had built the seed's sample.
Now I noticed that all those components are copied from src/Client. So, there are no "actual" duplicates. I can understand the need for a 'local' `node_modules` and separate build scripts etc.
So, yeah, my mistake, no need for a separate `shared` folder structure, as it is all shared from src/Client already.
